mlrun/runtimes/base.py

```diff
@@ -1086,11 +1131,15 @@ def _pre_deletion_runtime_resource_run_actions(
             )
             raise ValueError("Could not resolve run uid from runtime resource")
 
-        self._ensure_runtime_resource_run_status_updated(
-            db, db_session, project, uid, desired_run_state
+        logger.info(
+            "About to cleanup runtime resources, pre-deletion actions will be performed",
```

Comment:
```suggestion
            "Cleaning up runtime resources and running pre-deletion actions",
```
"About to" / "Starting to" ... same. needless fluff
Also are you sure this log is in the right place? this function doesn't have any calls to any "cleanup" or "pre-deletion" functions after that. just `_ensure_run_status_updated` and `_ensure_finished_run_logs_collected` which sounds very different (maybe just a naming issue)

Reply:
This log is inside the pre-deletion function, and the pre-deletion actions are exactly these 2, ensuring run status and ensuring logs collected.
changed to `"Performing pre-deletion actions before cleaning up runtime resources"`

Reply:
So log referring to cleanup should move to a function aware of the cleanup. It's not a huge sin but it's nicer that the logs don't exceed the scope of the logic. it's a better practice as you an move code around without having to change all the log lines because they all refer to where they are in the "grand scope" of things which, abstraction-wise, they shouldn't know about

Reply:
talked offline, leaving as is